### PR TITLE
Fix prelu bug in pytorch frontend

### PR DESCRIPTION
**Pull request by YuhengHuang42**
This PR tries to fix PReLU bug mentioned in https://github.com/apache/tvm/issues/8184
Reference: https://pytorch.org/docs/stable/generated/torch.nn.PReLU.html#torch.nn.PReLU

In short, there are two situations we need to consider:
1. When input dims < 2. PReLU in Pytorch can deal with this, while in TVM the default axis of `tvm.relay.nn.prelu` is set to 1, we will encounter an error when directly calling this function.
2. When input dims >= 2, input channel > 1 and num_parameters = 1. In this case will need to do a broadcasting on the alpha parameter.

@masahi  Please take a look at this PR, thanks!

### CODE REVIEW

**Comment by masahi** [comment]
Can you remove unrelated diffs in `test_forward.py`? My black version is `black, version 20.8b1`, running it on `test_forward.py` doesn't make any modification.
`

**Comment by YuhengHuang42** [comment]
> Can you remove unrelated diffs in `test_forward.py`? My black version is `black, version 20.8b1`, running it on `test_forward.py` doesn't make any modification.

The latest version should be OK. 

I encountered some weird bugs when using black, so I tried to use different versions of black to see the result. I will wait to see if this version can be successfully built or not.

**Review by masahi — APPROVED** [review]

**Comment by masahi** [comment]
thanks @YuhengHuang42
